### DISCUSSION TITLE
FX schematic undo

### DISCUSSION THREAD

**Discussion by age3rcm** [opentoonz/opentoonz#6551] [Ideas]
sometimes the click and drag fails and it gets out of bonds or in a place i do not want it to be

**Describe the solution you'd like**
make the moving of elements in the schematic view undo-able
![OpenToonz_Ay6QQTjgPq](https://user-images.githubusercontent.com/36406840/135883847-812fb899-9cd1-40b7-ab2f-3a9021356857.png)

**Comment by gab3d** [comment]
I just want sugest that if that undo ever happens to be, it'd be good if it is kept in its own stack, separated from the main undo.
